index.html

```diff
@@ -1,19 +1,129 @@
-<!doctype html>
+<!doctype html> 
 
 <html lang="en">
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Katie Fitzpatrick Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
 
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,900|Yantramanav:300,400,700" rel="stylesheet">
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+<div id=container>
+  <header class="navHeader"> 
+      <img class="profile" src="images/katieFitzpatrickLogo.png" alt="Katie Fitzpatrick Logo">
+    <nav> 
+      <a href="#">Work</a>
+      <a href="#">Kudos</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+    </nav>
+
+  </header>
+  <section class=intro>
+    <img class="profile" src="images/headshot3.png" alt="Portrait image of Katie Fitzpatrick.">
+    
+   
+    <div id="introAbout">
+        <h1>Katie Fitzpatrick</h1>
+        <h2> Product Manager</h2>
+        <p>Katie is a rare breed with the ability to seamlessly move and contribute to the technical, creative, and business arms of your organization. 
+            As a Product Manager for your organization you will find in Katie a Product Manager who is: </p>
+    
+    <ul class="unorderedLists">
+        <li>Collaborative and Commutative</li> 
+        <li>Innovative and Persuasive</li>
+        <li>Pragmatic and Methodical </li>  
+    </ul>
+    </div>
+    <div class=contact>
+        <h1>Contact Katie</h1>
+        <h2>Email:</h2>
+        <p>hellokatiefizzy@gmail.com</p>
+        <h2>Phone:</h2>
+        <p>828.335.5507</p>
+        <h2>City:</h2>
+        <p>Pittsburgh</p>
+    </div>
+  </section>
+  <h1 class ="sectionHead">Diverse Work Experience</h1>
+  <section class="timeline">
+    <div class="start">
+      <p>Start Here</p>
+    </div>
+    <div class="first">
```

Comment:
I think these items could reasonably be section tags instead of divs

Reply:
I could see that. This was my MVP version of the design. I figure at some point I may really want a resume site so eventually this is meant to look like a timeline. Just a side note to explain why I was naming classes that way - in case I wanted to use the timeline format in multiple places.